tfjs-backend-webgpu/src/index.ts

```diff
@@ -49,11 +49,12 @@ if (isWebGPUSupported()) {
     };
 
     if (supportTimeQuery) {
-      deviceDescriptor.requiredFeatures = ['timestamp-query' as const];
+      deviceDescriptor.requiredFeatures = ['timestamp-query' as const ];
     }
     const device: GPUDevice = await adapter.requestDevice(deviceDescriptor);
     return new WebGPUBackend(device);
   }, 3 /*priority*/);
 }
 
 export {webgpu};
+export * from './base';
```

Comment:
why not directly call `export * from './webgpu';` here?

Reply:
Refer it to webgl backend, we could put all wanted export utilities in base.ts file in future.

Reply:
The webgl backend puts kernels into the `register_all_kernels.ts` file and everything else into `base.ts`. This is for better readability, and makes sure any side effects from setting up the backend happen before kernels are registered (I'm not 100% sure this is necessary). If you want to do the same for webgpu, you could move everything from index.ts into base.ts. However, it's a style choice, and if you prefer to keep everything in index.ts, I think that's fine.